scripts/migrate/20200108-BOAC-3054/rds_pre_deploy_01_add_terms_in_attendance.sql

```diff
@@ -1,7 +1,36 @@
 BEGIN TRANSACTION;
 
-ALTER TABLE student.student_academic_status ADD COLUMN terms_in_attendance INT;
+CREATE TABLE IF NOT EXISTS student_annetest.student_academic_status_new
```

Comment:
@lyttam You probably want regular `student` schema, not `student_annetest`?

Reply:
@paulkerschen ah thanks! Amended.